.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-- '8'
-- '10'
+- '12.14.1'
```

Comment:
I think it is worth testing all the modules that we depend on for Mapeo with the specific versions of Node that we are shipping with the apps.

On mobile, this is currently `v10.13.0` but we will shortly upgrade to `v12.16.3`. On Desktop we are using Electron `v6.1.7` which runs main process code with Node `v12.4.0`.

I think it is useful to test these specific versions because it will catch any use of specific features which were introduced through subsequent minor releases, or catch us hitting any bugs that are specific to a version of Node.

TL;DR I think the versions we test on travis should be `v10.13.0`, `v12.4.0`, `v12.16.3` and `v12` (e.g. latest).

Reply:
I'm updating electron to 12.14.1 which is why I added that here, so perhaps we should have 10.13.0, 12.4.0, and 12.14.1.